visma/utils/integers.py

```diff
@@ -35,7 +35,8 @@ def factors(number):
     """
 
     factors = []
-    for i in range(1, abs(int(number)) + 1):
+    for i in range(1, abs(int(number**0.5)) + 1):
```

Comment:
Thanks for the PR. Can you make the following change? Since `number` is an integer,  it's sqrt can be complex which can lead to _complex to int typeerror_. 
```suggestion
    for i in range(1, int(abs(number)**0.5) + 1):
```
Run the modules test locally (`./run test modules`) before committing and make changes in the corresponding tests module, if necessary. You can refer [this](https://github.com/aerospaceresearch/visma/blob/master/CONTRIBUTING.md#make-sure-all-tests-pass-before-making-a-pr) to know more about running tests.

Reply:
yes i did the changes suggested by you but now im getting the assertion error, please have look at this.
![screenshot 8](https://user-images.githubusercontent.com/44905974/51083213-2489d480-173c-11e9-9de5-feaac3dd7347.png)
expected output is same as the value evaluated by factorize function , only difference is (x+1) term is written before (x-1) .
can you help me what can i do about this?

Reply:
Since the change from _max_ to _min_ has been made in the `gcd()`, it is being reflected here. You can change the output in `test_factorize()` to `(x+1)*(x-1)*...` and run the tests again.

Reply:
I think you forgot to commit [this](https://github.com/aerospaceresearch/visma/pull/102#discussion_r247323383) change.